Userland/Applications/CertificateSettings/CertificateStore.cpp

```diff
@@ -59,6 +68,43 @@ GUI::Variant CertificateStoreModel::data(GUI::ModelIndex const& index, GUI::Mode
     return {};
 }
 
+ErrorOr<size_t> CertificateStoreModel::try_add(Vector<Certificate> const& certificates)
+{
+    auto size = m_certificates.size();
+    TRY(m_certificates.try_extend(certificates));
+    return m_certificates.size() - size;
+}
+
+ErrorOr<void> CertificateStoreWidget::try_import()
+{
+    auto file = FileSystemAccessClient::Client::the().open_file(window(), "Choose PEM to import...");
```

Comment:
If you really want to ignore filesystem open errors here, and not pass them to the caller, then the pattern is more like this
```
auto file_or_error = FSAC::open(...);
if (file_or_error.is_error())
    return {}; // Swallow FSAS errors, they were already reported(?)
auto file = file_or_error.release_value();

// ... use file from now on.
```

Reply:
The only FSAC error I was able to get at this point was -1 for pressing cancel. If there are more, they might need to be reported, but I never saw any.

Will see about renaming the variables a bit, but the thing you get from `release_value()` is not a `Core::File` but still something FSAC specific.